### ISSUE TITLE
cmd/gocritic: rangeValCopy/rangeExprCopy triggers in test helpers

### ISSUE THREAD

**Issue by Quasilyte** [go-critic/go-critic#581]
There is a filter for `Test*` functions in performance checkers that should reduce false-positives when checking test files.
Turns out this is not enough as it will complain about function being called from the `Test*` functions, like test helpers.

**Comment by Quasilyte** [comment]
CC @ernado

**Comment by ernado** [comment]
Thank you!

The following function triggers rangeValCopy:
```go
func ensureNoErrors(t *testing.T, logs *observer.ObservedLogs) {
	for _, e := range logs.TakeAll() {
		if e.Level == zapcore.ErrorLevel {
			t.Error(e.Message)
		}
	}
}
```

Current workaround: I've move it into separate `testutil` package and ignored via `--exclude`
